umbraco-heartcore/release-notes/2024-08-releasenotes.md

```diff
@@ -0,0 +1,14 @@
+# August 2024
+The following changes have been released to all Heartcore sites.
+
+## Webhook Enhancements
+With the core CMS having added webhook support, we have adapted the same function in Heartcore to use the same user interface. You'll now get the same consistent editing experience in both core Umbraco and Heartcore, but also some new features!
```

Comment:
We try to not use third person (we). Also, I've rewritten this a bit according to Grammarly recommendations.
Let me know if something doesn't make sense!

```suggestion
With the core CMS adding webhook support, the same functionality has been adapted into Heartcore to use the same user interface. You will now get the same consistent editing experience in both Umbraco CMS and Umbraco Heartcore, and it also comes with new features.
```